### PR TITLE
Added File: Bootstrap Assignment-1

### PR DESCRIPTION
**Pull request by akash19972306**

### CODE REVIEW

File: akash1/Bootstrap/ASSIn1/assignnmentb1.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>MI-MEAN-11</title>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <script src="https://kit.fontawesome.com/15329e9b1c.js" crossorigin="anonymous"></script>
+
```
Comment (sourabh513) [inline akash1/Bootstrap/ASSIn1/assignnmentb1.html]: remove extra line

File: akash1/Bootstrap/ASSIn1/assignnmentb1.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+                                    <path
+                                        d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"
+                                    />
+                                    <path
+                                        d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"
+                                    />
+                                </svg>
+                                &nbsp;&nbsp;Snehal Ghodmare posted a new material: Bootstrap-Document<br /><span style="color: grey">
+                                    &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp; &nbsp;&nbsp;Dec 16</span
+                                >
```
Comment (sourabh513) [inline akash1/Bootstrap/ASSIn1/assignnmentb1.html]: wrong indentation

File: akash1/Bootstrap/ASSIn1/assignnmentb1.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+                                    <path
+                                        d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"
+                                    />
+                                    <path
+                                        d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"
+                                    />
+                                </svg>
+                                &nbsp;&nbsp; Snehal Ghodmare posted a new assignment: Evaluation 12Dec20<br /><span style="color: grey">
+                                    &nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;Dec 12(Edited Dec 12)</span
+                                >
```
Comment (sourabh513) [inline akash1/Bootstrap/ASSIn1/assignnmentb1.html]: wrong indentation

File: akash1/Bootstrap/ASSIn1/assignnmentb1.html [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+                                    <path
+                                        d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"
+                                    />
+                                    <path
+                                        d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"
+                                    />
+                                </svg>
+                                &nbsp;&nbsp; Snehal Ghodmare posted a new material: Updated Document for HTML, CSS Content<br /><span style="color: grey"
+                                    >&nbsp;&nbsp; &nbsp;&nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;Dec 10</span
+                                >
```
Comment (sourabh513) [inline akash1/Bootstrap/ASSIn1/assignnmentb1.html]: wrong indentation

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**